Update draft-kuehlewind-update-tag.md (#18)
mirjak/draft-kuehlewind-update-tag: draft-kuehlewind-update-tag.md

## Formating Style of Amendments

Currently some RFCs use and OLD/NEW style to highlight actual text changes others

> simply describe the changes in text. This document recommends the use of the 

Reviewer: ```suggestion
simply describe the changes in text. While this document does not require a specific format of amendments,
it recommends the use of the 
```

Author: Looks good. Replace "does no" with "does not"

> OLD/NEW style in Amending RFCs for minor and limited number of changes. This makes 
> the changes unambiguous and specific. It also potentially enables automated tools 
> to consume these changes and be able to produce a marked up copy of the Amended RFC 
> that shows the effect of these changes in place. If extensive or large number of

Reviewer: ```suggestion
OLD/NEW style in Amending RFCs for minor and limited number of changes. This could 
enable the use of automated tools in the future to produce a marked up copy of the Amended
RFC that shows the effect of these changes in place. If extensive or a large number of
```

Author: Looks good. Change "in future enable the use of automated tools" to "enable the use of automated tools in the future" for better readability.
